apps/src/weblab2/helpers/aiTutorStructuredResponseHelper.ts

```diff
@@ -40,7 +43,7 @@ const getAnswerJsonSchema = (): JsonObjectSchema => {
           additionalProperties: false,
         },
         description:
-          '`html` and/or `css` fences. When providing modifications to student code, provide the entire contents of the file. The list can be empty. Code should be formatted with appropriate newlines and indentation. The student will need to copy and paste this code into their project.',
+          '`html`, `css`, or `js` fences. Limit to one language (html, css, or js) across the entire list. When providing modifications to student code, provide the entire contents of the file. The list can be empty. Code should be formatted with appropriate newlines and indentation. The student will need to copy and paste this code into their project.',
```

Comment:
fyi @fisher-alice I just changed the text here, I found a case where it was returning both html & css and made the limiting more explicit.